templates/vue/src/App.vue

```diff
@@ -61,6 +63,12 @@ export default {
     },
   },
   mounted() {
+    client.getTheme().then(res => {
```

Comment:
perhaps you can put this into the `data `variable on line 84 and the following code inside that `then` statement as well